clippy.toml

```diff
@@ -0,0 +1 @@
+avoid-breaking-exported-api = false
```

Comment:
Why?

Reply:
This is a library and clippy prevents hints that change the exposed API in order to prevent breaking changes. But it also doesn’t lint on additions that haven’t been released so far.
This disables that, therefore it hints on everything (like with binary projects) again.

Especially on a 0. project a good API is more important than to prevent breaking changes.

Reply:
Why is that in its own file though? Whats different with this lint setting that it is not with the other lint settings in the cargo.toml?

Reply:
Probably for historic reasons. There are lints and there are lint configurations. A common configuration of rules is the MSRV which is auto-detected. But it can also be overridden with this file.
Lints can be enabled with `#![warn(…)]` for a crate, and it's not that long that the lints can also be set in the `Cargo.toml`. But it's the rules there, not their configuration values.